src/stringMe.js

```diff
@@ -0,0 +1,26 @@
+export default function stringMe(string){
+  let currentLetter = string[0]
+  let subString = []
+  let subStrings = []
+  for(let i = 0; i < string.length; i++){
+    if(string[i] === currentLetter){
+      subString.push(string[i])
+    }
+    if(string[i] != currentLetter){
+      subStrings.push(subString)
+      subString = [string[i]]
+      currentLetter = string[i]
+    }
+  }
+  subStrings.push(subString)
+
+  let longestPair = 0
+  for(let i = 0; i < subStrings.length - 1; i++){
+    if(subStrings[i].length + subStrings[i + 1].length
+      > subStrings[longestPair].length + subStrings[longestPair + 1].length){
+
+      longestPair = i
+    }
+  }
```

Comment:
Overall I like how simple and readable this solution is. One test it would fail is a word like Mississippi: the longest string with only two unique characters would be ississi, but this code produces iss. 

Rebuilding the sub string is going to be a little more complicated than comparing adjacent array lengths, you'll have to iterate through them again and compare the current and previous values to the next one to get the actual length, and then store the value of the current longest sub string.